### PR TITLE
Add GraalVM CE 19.0.2

### PR DESCRIPTION
**Pull request by wololock**
Release notes: https://www.graalvm.org/docs/release-notes/#1902

GitHub release information: https://github.com/oracle/graal/releases/tag/vm-19.0.2

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 1]
```diff
@@ -305,4 +305,26 @@ class JavaMigrations {
...
   }
+
+  @ChangeSet(order = "144", id = "144-add_graalvm_19_0_2", author = "wololock")
+  def migrate144(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        candidate = "java",
+        version = "19.0.2-grl",
+        url = "https://github.com/oracle/graal/releases/download/vm-19.0.2/graalvm-ce-linux-amd64-19.0.2.tar.gz",
+        platform = Linux64),
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: We now require a vendor to be set for all Java versions. You can add an extra param to version of `vendor = Some(Graal)`.

Comment (wololock) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Thanks, @marc0der, for explaining that! Pushed changes including the vendor.

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Awesome, thanks!

**Comment by marc0der** [comment]
Thanks for contributing again!

**Pull request merged**